mill/__init__.py

```diff
@@ -380,6 +388,24 @@ def sync_heater_control(self, device_id, fan_status=None,
                    "value": int(set_temp),
                    "key": "holidayTemp"}
         await self.request("changeDeviceInfo", payload)
+        heater = self.heaters.get(device_id)
+        heater.set_temp = set_temp
+
+        """If heater is associated with a room override room settings"""
+        if heater.room is not None:
+            await self.set_heater_room_override(device_id, 1)
```

Comment:
Why would we change between room control and individual control?

Reply:
I think we should not do that automatically.

Reply:
> Why would we change between room control and individual control?

Testing I found that after setting temperature for a heater, that heater would not respond when I changed temperature for the room. To get the heater to update to room temperature I had to toggle the individual control switch in the app to on and then back to off.
Also this is the way it's handled by the Mill app. You cannot set temperature for a heater without setting it to individual control first.

Reply:
> I think we should not do that automatically.

Maybe not, but then we need to find another way to define whether the heater should follow the room settings or not.
The logic behind setting it automatically is "your last action rules". I believe when changing the temperature of a room you want that temperature without concerning yourself about the individual heaters.
If you really need to control one heater manually it could be removed from the room and treated as an independent heater.

Reply:
We should not automatically change between room control and individual control. But keep it as an option, like in the mill app.

Reply:
I agree this would be the prefrable solution. We should try to mimic the behavour of the Mill app as far as we can.
Do you have any thoughts on how this can accomplished with the climate component in HA?
I will take a closer look at the climate component as soon as I have som hours to spare...

Reply:
What about this approach:
We enable SUPPORT_AUX_HEAT for all room-attached heaters with is_aux_heat()=true meaning the heater is set to individual control.

The AUX_HEAT attribute can be altered in the thermometer card giving the user the ability to turn individual control on or off.

If a user changes a room-attached heater we automatically set it to individual control.

We add an optional parameter under Mill in configuration.yaml - force_room_control. If this parameter is set to true changing a room will automatically turn off individual control for all heaters in the room.